tests/js/web/PulsarUIComponentTest.js

```diff
@@ -7,6 +7,8 @@ var $ = require('jquery'),
     tab = require('../../../js/libs/tab'),
     PulsarUIComponent = require('../../../js/PulsarUIComponent');
 
+    $.fx.off = !$.fx.off;
```

Comment:
Can we explicitly set this to `true` or `false`? Toggling the value assuming it will always be one or the other seems a bit risky.